webui/react/src/e2e/models/components/ProjectCreateModal.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseComponent } from 'e2e/models/BaseComponent';
+import { Modal } from 'e2e/models/hew/Modal';
+
+/**
+ * Returns a representation of the Workspace create/edit modal component.
+ * This constructor represents the contents in src/components/Page.tsx.
+ * @param {object} obj
+ * @param {implementsGetLocator} obj.parent - The parent used to locate this Page
```

Comment:
docstring

Reply:
maybe we could replace this line with `used to locate` so we dont have to "get this right" each time